docs/bulk_api.md

```diff
@@ -0,0 +1,39 @@
+# Bulk API Overview
+
+Bulk API endpoints allows to perform bulk operations in single web request. There are currently following bulk api actions:
+- /api/v2/bulk/job_launch
+- /api/v2/bulk/host_create
+
+## Bulk Job Launch
+
+Provides feature in the API that allows a single web request to achieve multiple job launches. It creates a workflow job with individual jobs as nodes within the workflow job. It also supports providing promptable fields like inventory, credential etc.
+
+Following is an example of a post request at the /api/v2/bulk/job_launch
+
+```commandline
+{
+"name": "Bulk Job Launch",
+"jobs": [
+   {"unified_job_template": 7, "identifier":"foo", "limit": "kansas", "credentials": [1]},
+   {"unified_job_template": 8, "identifier":"bar", "inventory": 1, "execution_environment": 3},
+   {"unified_job_template": 9}
+]
+}
+```
+
+The above will launch a workflow job with 3 nodes in it. 
```

Comment:
Could you add something to document the use of top-level parameters? Like if `inventory` was provided as a top-level key. I think the feature is that it will place those as prompts on the workflow job. But that's from somewhat indirect evidence from the tests, I am still not clear on the intent.

Reply:
Done. Added an  example for that